test/Microsoft.AspNet.Mvc.Razor.Host.Test/MvcRazorParserTest.cs

```diff
@@ -15,30 +16,132 @@ namespace Microsoft.AspNet.Mvc.Razor
 {
     public class MvcRazorCodeParserTest
```

Comment:
not seeing a test that includes two `@tagHelperPrefix` directives in a single `CodeTree` aka `_GlobalImport.cshtml` file.  where is that error case tested?